source/_components/repetier.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Repetier-Server Component
+description: "Instructions how to add Repetier-Server sensors to Home Assistant."
+date: 2019-03-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+featured: false
+ha_release: 0.88
+ha_iot_class: "Local Polling"
+---
+
+[Repetier-Server](https://www.repetier-server.com/) is a 3D printer/CNC server, able to control multiple devices on the same server.
+This component handles the main integration to the server.
+
```

Comment:
```suggestion
There is currently support for the following device types within Home Assistant:

- Sensor
```

Reply:
In which path do I put the image for the logo?
Can't seem to find them anywhere.